go/vt/sqlparser/sql.y

```diff
@@ -7570,6 +7584,7 @@ non_reserved_keyword:
 | AVG %prec FUNCTION_CALL_NON_KEYWORD
 | AVG_ROW_LENGTH
 | BEGIN
+| BEFORE
```

Comment:
`BEFORE` should come before `BEGIN` in alphabetic order

Reply:
fixed